src/main/java/io/netty/incubator/codec/quic/QuicConnectionEvent.java

```diff
@@ -18,14 +18,14 @@
 import java.net.SocketAddress;
 
 /**
- * {@link QuicEvent} which is fired when an QUIC connection migration was detected.
+ * {@link QuicEvent} which is fired when an QUIC connection creation or migration was detected.
  */
-public final class QuicConnectionMigrationEvent implements QuicEvent {
+public final class QuicConnectionEvent implements QuicEvent {
 
     private final SocketAddress from;
     private final SocketAddress to;
 
-    QuicConnectionMigrationEvent(SocketAddress from, SocketAddress to) {
+    QuicConnectionEvent(SocketAddress from, SocketAddress to) {
```

Comment:
Also lets rename the methods to make things a bit less confusing:

`oldAddress()`, `newAddress()` and add something to the javadocs for `oldAddress()` to tell that this might be `null`